installer/scripts/bootstrap.bash

```diff
@@ -38,6 +38,7 @@ fi
 
 # Source the installer commands
 source ~/.tue/setup.bash
+echo "[bootstrap] Loading tue-env"
```

Comment:
This hardly takes any time, so unnecessary in my opinion. And if you put it there, put it before sourcing it.